tests/integration/test_cubepart.py

```diff
@@ -858,6 +865,20 @@ def it_provides_stddev_measure_for_cat_x_mr(self):
         )
         assert slice_.table_base.tolist() == [3, 3, 3]
 
+    def it_provides_share_of_sum_measure_for_mr_x_mr(self):
+        slice_ = Cube(CR.MR_X_MR_SUM).partitions[0]
+
+        assert slice_.share_sum == pytest.approx(
+            np.array(
+                [
+                    [np.nan, np.nan, np.nan],
```

Comment:
urgh, this is ugly... why do we have this ugly expectation? couldn't we find something more useful?